crates/hyperswitch_connectors/src/connectors/aci/transformers.rs

```diff
@@ -1581,3 +1613,9 @@ pub struct AciWebhookNotification {
     pub action: Option<AciWebhookAction>,
     pub payload: serde_json::Value,
 }
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub struct AciMandateMetadata {
+    pub initial_transaction_id: String,
+}
```

Comment:
Could you please clarify the purpose of the `initial_transaction_id` field and the requirement it is intended to address?